packages/ui-components/src/__tests__/InputRegistryUrl.test.ts

```
@@ -2,41 +2,41 @@ import { render, screen, fireEvent } from '@testing-library/svelte';
+
+export const initialRegistry: Partial<RegistryManager> = {
+	getCurrentRegistry: vi.fn(() => mockCurrentRegistry ?? mockDefaultRegistry),
+	setRegistry: vi.fn((newRegistry: string) => {
+		mockCurrentRegistry = newRegistry;
+	}),
+	resetToDefault: vi.fn(() => {
+		mockCurrentRegistry = mockDefaultRegistry;
+	}),
+	updateUrlWithRegistry: vi.fn(),
+	isCustomRegistry: vi.fn(() => mockCurrentRegistry !== mockDefaultRegistry)
+};
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)